.github/workflows/code-coverage-and-coveralls.yml

```
@@ -28,6 +28,9 @@ jobs:
       with:
         php-version: '7.4'
         coverage: xdebug
+        github-token: ${{ secrets.PUBLIC_REPO_ACCESS }}
```

actionlint (1.7.7)

input "github-token" is not defined in action "shivammathur/setup-php@v2". available inputs are "coverage", "extensions", "ini-file", "ini-values", "php-version", "php-version-file", "tools"

(action)